lib/community/bloc/community_bloc.dart

```diff
@@ -70,6 +70,14 @@ class CommunityBloc extends Bloc<CommunityEvent, CommunityState> {
         try {
           CommunityView communityView = await followCommunity(event.communityId, event.value);
 
+          if (GlobalContext.context.mounted) {
+            if (event.value) {
+              showSnackbar(AppLocalizations.of(GlobalContext.context)!.subscriptionRequestSent);
+            } else {
+              showSnackbar(AppLocalizations.of(GlobalContext.context)!.unsubscriptionRequestSent);
```

Comment:
I'm not sure we need this one since I don't believe unsubscripions have the same delay as subscriptions where the can be in `pending` for a while. But I'll leave that up to you!

Reply:
I'm not actually too sure how the pending states work in this situation, so I just applied it to both! I can remove the unsubscription one though since you may be correct here!

If you know of a way to trigger the pending state, let me know and I can test it out